share/lib/common.sh

```diff
@@ -27,10 +27,10 @@ service_action() {
   service="$1"
   action="$2"
 
-  if [ -x /usr/sbin/service ]; then
-    CMD="service ${service} ${action}"
-  elif [ -x /etc/init.d/${service} ]; then
+  if [ -x /etc/init.d/${service} ]; then
```

Comment:
list-units always returns 0, the pattern is just a filter and an empty result is not considered as an error.